Unity3D/Assets/RosSharp/Scripts/ImageManager.cs

```diff
@@ -1,4 +1,24 @@
-﻿using RosSharp.RosBridgeClient;
+﻿/*
+© Federal Univerity of Minas Gerais (Brazil), 2017
+Author: Lucas Coelho Figueiredo (me@lucascoelho.net)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+<http://www.apache.org/licenses/LICENSE-2.0>.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
For better readability, I suggest less empty lines. 
Adding one or max. 2 empty line where approriate  between 2 methods, after header, etc.) is sufficient.